media/js/firefox/australis/tour.js

```diff
@@ -37,7 +40,11 @@
             }
 
             // Push custom GA variable to track Sync visibility
-            gaTrack(['_setCustomVar', 6, 'Sync Visible', visible, 2]);
+            window.dataLayer = window.dataLayer || [];
```

Comment:
If doing this check, should only happen once per file (probably at the top of the IIFE, just after `'use strict';`).